joda-time/src/main/java/org/joda/time/field/PreciseDateTimeField.java

```diff
@@ -77,13 +77,14 @@ public PreciseDateTimeField(DateTimeFieldType type,
      * @return the amount of fractional units extracted from the input.
      */
     @Override
-    public int get(long instant) {
-        if (instant >= 0) {
-            return (int) ((instant / getUnitMillis()) % iRange);
-        } else {
-            return iRange - 1 + (int) (((instant + 1) / getUnitMillis()) % iRange);
+        public int get(long instant) {
+            if (instant >= 0) {
+                return (int)((instant / getUnitMillis()) % iRange);        
+            } 
```

Comment:
Does the other code in this file have `else`? Does it look like `} else {`, or is it really split into several lines?

Reply:
There were no else's in this file, or other code that has an if return structure, however I looked around some of the files and saw that in the files I looked in the code that had an if return structure did not have an else so I removed it.